tendrl/node_agent/flows/flow_execution_exception.py

```diff
@@ -0,0 +1,4 @@
+class FlowExecutionFailedError(Exception):
```

Comment:
Remove this file from the PR, since we just want to merge the framework bits and not the ansible task related bits